src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -1272,6 +1272,9 @@ static int client_timeout_resend(
                 next_timeout = client_compute_request_timeout(time_now, client->attempt);
                 break;
 
+        default:
```

Comment:
Why adding this above the case below? Normally the default is the last label

Reply:
Because I thought that assertions can be disabled (with `NDEBUG`) and I would have to handle the case still after the assertion. Hence the `fall-through`.  But actually, `assert_not_reached()` cannot be disabled with `NDEBUG` and it has a `_noreturn_` attribute. So, the compiler always knows that this function doesn't return.

Changed.